src/overlays/actors/ovl_Eff_Stk/z_eff_stk.c

```diff
@@ -28,16 +28,81 @@ ActorInit Eff_Stk_InitVars = {
     (ActorFunc)EffStk_Draw,
 };
 
-#endif
+void func_80BF0DE0(EffStk* this, PlayState* play);
```

Comment:
Could you put this function declaration closer to the other declarations (below `EffStk_Draw` but with an extra line in between) i.e.

```c
void EffStk_Draw(Actor* thisx, PlayState* play);

void func_80BF0DE0(EffStk* this, PlayState* play);